examples/sdk-playground/packages/client/src/pages/CurrentGuildMember.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import discordSdk from '../discordSdk';
+import ReactJsonView from '../components/ReactJsonView';
+import {useLocation} from 'react-router-dom';
+import {EventPayloadData} from '@discord/embedded-app-sdk';
+
+export default function CurrentGuildMember() {
+  const [currentGuildMember, setCurrentGuildMember] =
+    React.useState<EventPayloadData<'CURRENT_GUILD_MEMBER_UPDATE'> | null>(null);
+  const location = useLocation();
+
+  React.useEffect(() => {
+    const {channelId} = discordSdk;
+    if (!channelId) return;
+
+    const handleCurrentGuildMemberUpdate = (
+      currentGuildMemberEvent: EventPayloadData<'CURRENT_GUILD_MEMBER_UPDATE'>,
+    ) => {
+      setCurrentGuildMember(currentGuildMemberEvent);
+    };
+
+    const guildId = discordSdk.guildId;
+    if (guildId) {
+      discordSdk.subscribe('CURRENT_GUILD_MEMBER_UPDATE', handleCurrentGuildMemberUpdate, {
+        guild_id: guildId,
```

Comment:
Same question about if this arg should be guild-based or channel-based. Pretty trivial to switch over either way

Reply:
Per other PR looks g2g with guild id 👍🏻